pkg/kv/kvserver/batcheval/cmd_end_transaction.go

```diff
@@ -1243,6 +1243,20 @@ func splitTrigger(
 	return splitTriggerHelper(ctx, rec, batch, in, h, split, ts)
 }
 
+// TestingSplitTrigger is a wrapper around splitTrigger that is exported for
+// testing purposes.
+func TestingSplitTrigger(
```

Comment:
Any reason not to just export `splitTrigger` func? Or you plan to diverge this duplicate in some way?